templates/index.html

```diff
@@ -70,7 +70,7 @@ <h1><img src="static/image/logo.png" style="width: 200px;" /> botamusique Web In
                             </button>
 
                             <input type="range" class="custom-range" id="volume-slider" 
-                                    min="0" max="100" step="1" value="50"
+                                    min="0" max="1" step="0.01" value="0.5"
                                     onchange="request('post', {action : 'volume_set_value', new_volume : this.value})">
```

Comment:
When adjusting the value with the mouse wheel by scrolling while it is focused, or the left/right arrow keys, a lot of network events get triggered.
If this is a problem worth fixing we could move the request that is triggered in the onchange event into a function that checks when the last update was sent and limit it to a certain frequency.

Reply:
Good insight. I think you can try use setTimeout to solve this. Just avoid sending requests once onchange is triggered. Put 
```
clearTimeout(request_to_send);
request_to_send = setTimeout(you_function_to_send_request, 300);
```
in it.

Reply:
That is a good idea, I will read a bit more about those functions and think about edge cases before implementing it.
I am a bit worried about creating a loop where multiple clients cause circular updates when each tries to adjust the volume and send delays are integrated.

Reply:
Hi, @azlux and I are preparing for a new release in this weekend. Do you think you can make all changes in this weekend so we can include your smart design in this release?

Reply:
Or maybe a 
```
<input type="range" class="custom-range" id="volume-slider" min="0" max="1" step="0.01" value="0.5" onkeydown="event.preventDefault()" onchange="request('post', {action : 'volume_set_value', new_volume : this.value})">
```
to avoid keyboard input.